src/web/forms/fields/AttachmentViewer/index.js

```diff
@@ -177,6 +177,7 @@ class AttachmentViewerComponent extends Component {
           return (
             <div key={m.id} className="column is-narrow">
               {content}
+              <div>{m.name}</div>
```

Comment:
Can you please handle the case where the filename is too long?
<img width="1068" alt="2018-09-02 22 40 49" src="https://user-images.githubusercontent.com/3135397/44956591-5a52bd00-af01-11e8-821f-69d02e366f25.png">

I think it's good to insert `...` into the middle of filename like `TOO_LO...AME.png`

Reply:
Oh! Got it, will do :)

Reply:
Please pull newest changes & take a look